packages/asc-ui/src/components/TopTaskButton/TopTaskButtonStyle.ts

```diff
@@ -0,0 +1,35 @@
+import styled from '@datapunt/asc-core'
+import { svgFill, themeColor, themeSpacing } from '../../utils'
+import IconStyle from '../Icon/IconStyle'
+import Paragraph from '../Paragraph'
+
+export default styled.a<{
```

Comment:
Why an anchor and not a button?

Reply:
Since the TopTaskButton is always used as a navigational element we decided to use an anchor element.

Reply:
Ok, so why not rename it to TopTaskLink? Button is very confusing if it's not a button but a link..

Reply:
I'll propose it at the design system meeting, the terminology is taken from there.